packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -97,6 +97,38 @@ describe('cluster', () => {
     });
   });
 
+
```

Comment:
No need for this extra empty line (there should be a single empty line between the tests).

Reply:
Still not addressed.